src/DAO/UserDAOImpl.java

```diff
@@ -0,0 +1,82 @@
+package DAO;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import Model.Constant;
+import Model.User;
+import Utility.*;
+public class UserDAOImpl implements UserDAO {
+	private Connection dbConnection = DBConnection.getInstance().getConnection();
+	PreparedStatement statement=null;
+	ResultSet resultset=null;
+	@Override
+	public boolean insertUser(User user) {
+		Connection connection=DBConnection.getInstance().getConnection();
+		String name=user.getName();
+		String password=user.getPassword();
+		String username=user.getUsername();
+		String country=user.getCountry();
+		try {
+
+			statement = connection.prepareStatement(Constant.INSER_USER);
+			statement.setString (1, name );
+			statement.setString(2,  password);
+			statement.setString(3, username); 
+			statement.setString(4, country);
+			statement.execute();
+
+			System.out.println("successful");
+
+		} catch (SQLException e) {
+			// TODO Auto-generated catch block
+			System.out.println("something wrong" + e);
+		}
+
+		return true;
+
+
+	}
+	public int userLogin(User user) {
+		Connection connection=DBConnection.getInstance().getConnection();
```

Comment:
You could make connection as object variable. Avoiding repetitive code is highly appreciated.